terraform/gcp/best-practice/gcp-kms-rotation-period.yaml

```diff
@@ -0,0 +1,22 @@
+rules:
+- id: gcp-kms-rotation-period
+  patterns:
+    - pattern: resource
+    - pattern-inside: |
+          resource "google_kms_crypto_key" "..." {
+            ...
+            rotation_period = "$ROTATION"
+            ...
+          }
+    - metavariable-comparison:
+        metavariable: $ROTATION
+        comparison: $ROTATION > 7776000

```

Comment:
I assume this is failing because the test cases have an `s` string at the end. Let me see if there's anything that helps process strings in `metavariable-comparison` that might be undocumented.

Reply:
I've noodled on this for a bit but I can't figure out any way to do the comparison successfully. I've pointed this out to the Semgrep folks so it might require a new feature or modification to an existing one to make work.

In the meantime, what if we took this rule out from this PR, made an individual PR with it, and then moved forward with the rest of the rules?

Reply:
Yes, sounds good to me!

Reply:
Editor link for preserving the rule:https://semgrep.dev/playground/s/wwbB